content/4_Gold/Combinatorics.problems.json

```diff
@@ -30,21 +30,6 @@
       }
     }
   ],
-  "sample3": [
-    {
-      "uniqueId": "skyline",
```

Comment:
are you not going to add this problem to the new module?

Reply:
Well I really don't know how to demonstrate why catalan number is the answer.

Reply:
Alright I added it as a practice problem.